infra/terraform/modules/codeowners_file/main.tf

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "github" {
+  owner = var.org
+}
```

Comment:
nit: I think provider initialization within modules are deprecated. We can initialize in root and pass to module too
https://developer.hashicorp.com/terraform/language/modules/develop/providers#passing-providers-explicitly

Reply:
Sure, moved to the new model.